Dockerfile.flask

```diff
@@ -1,44 +1,17 @@
-FROM ubuntu:20.04
+FROM python:3.11-alpine

```

Comment:
I think it makes sense to rename this file to `Dockerfile.dev` and change this in `docker-compose.yaml`, to make it more clear how this Dockerfile is used.

Reply:
good idea, done :)